src/controls/qml/qmldir

```diff
@@ -33,6 +33,7 @@ TextField 1.0 TextField.qml
 ProgressBar 1.0 ProgressBar.qml
 ActionButton 1.0 ActionButton.qml
 NemoIcon 1.0 NemoIcon.qml
+Wallpaper 1.0 Wallpaper.qml
```

Comment:
no needed

Reply:
file:///usr/lib/qt/qml/QtQuick/Controls/Nemo/ApplicationWindow.qml:192:5: Wallpaper is not a type